File: testebackendestagiov2/src/main/java/io/github/pedrobicudo/testebackendestagiov2/model/domain/repositories/EquipmentStateRepository.java
package io.github.pedrobicudo.testebackendestagiov2.model.domain.repositories;

import io.github.pedrobicudo.testebackendestagiov2.model.domain.entities.EquipmentState;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface EquipmentStateRepository extends JpaRepository<EquipmentState, UUID> {
}

File: EquipmentApi/EquipmentApi/Data/Repositories/EquipmentStateRepository.cs
using EquipmentApi.Data.Interfaces;
using EquipmentApi.Entities;

namespace EquipmentApi.Data.Repositories
{
    public class EquipmentStateRepository : BaseRepository<EquipmentState>, IEquipmentStateRepository
    {
        public EquipmentStateRepository(EquipmentDbContext context) : base(context)
        {
        }
    }
}
